### ISSUE TITLE
Show Details performance issue: don't display all seasons by default [feature request]

### ISSUE THREAD

**Issue by kailia** [SiCKRAGETV/sickrage-issues#3195]
Hello, opening or Refreshing from Disk, or Full Updating a Show page for a show with many seasons (e.g. Daily Show, SNL, National Geographic, etc...) can take a very, very long time and a lot of CPU, and freeze your browser (especially firefox) for 15 or 20 seconds, sometimes more. I have a feeling this could be avoided simply by e.g. only displaying the last 5 seasons by default, and having a button on the page to go to the remaining seasons or to display All seasons. A setting in Options, General, Interface could enabling toggling this features on and off. This way, displaying such shows shouldn't have such a great impact.

P.S. thanks for making such a great program.

**Comment by SiCKRAGEB0T** [comment]
## Thanks for the bug/issue report
Please make sure to submit all of the [required information](https://github.com/SiCKRAGETV/sickrage-issues/blob/master/README.md), or we won't be able to help you!
Provide as **much** information as you can such as debug logs, commit hash, and tell us how to replicate the problem.For basic **questions** use the [ [forum] ](https://sickrage.tv/forums/), and for **Feature Requests** [ [Feathub] ](http://feathub.com/SiCKRAGETV/SickRage)
Also make sure the title accurately describes your issue. ("SR not working" etc. is not ok!)

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist)

## Many issues can simply be solved by:
- Making sure you update to the latest version.
- Have you tried turning your device off and on again?
- Using the **search** function to see if this issue has already been reported/solved.
- Do a [GIT reset](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#update-problems-try-this).
- Checking the [ [Wiki] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki) for [ [Guides] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Installation-&-Configuration-Guides), [ [FAQS] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes) [ [Main settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Settings-explained), [ [Show settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Show-settings-explained) [ [Remaining settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Remaining-settings-explained)
- Shutting down SickRage and removing the `cache/mako` & `cache/sessions` folders.
- If SickRage doesnt find any episodes you probably need to :
 - add a [Scene Exception](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Scene-exceptions-and-numbering)
 - Set the [Default Episode Status](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#newly-aired-shows-are-not-downloading-and-set-to-skippedignored) to `WANTED`
 - Make sure you understand how [quality detection](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Quality-Settings#quality-detectiondetermination) works during searches.
 - Shutdown SickRage and remove the `cache.db` file.
- If you use a QNAP and have SSL errors try this [package](http://apps.qnap.community/11-community/17-qsickrage).
- If you have a `'ascii' codec can't encode character` error, than set your Locale to [UTF-8](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#i-have-problems-with-special-characters-%C3%A9-etc-what-can-i-do)

*ps. If you enjoy Sickrage and like the support provided, than please consider making a [donation](https://github.com/SiCKRAGETV/SickRage/wiki/Donations) to keep SickRage alive.*

**Comment by miigotu** [comment]
Feature requests go on feathub, not the issue tracker. Check the bot reply for the link

**Comment by fernandog** [comment]
We already have that. Check general config

**Comment by kailia** [comment]
Well, in that case it doesn't work. Whatever I do, all seasons are always shown.

**Comment by fernandog** [comment]
What is the name of the setting you enabled?

**Comment by kailia** [comment]
CORRECTION: I closed and reopened my (firefox) browser, and now I see the difference.
But it doesn't make as such of a difference in processing time as one could hope.

I did the following tests while running a "Re-scan files" on The Daily show (stored locally on SSD, just 3 episodes actually present), just to keep the CPU a bit busy:
- Refresh (F5) the Daily Show page with "show all seasons" option ON: 30 to 58 seconds
- id. with "show all seasons" option OFF: 18 to 30 seconds
- Refresh (F5) the Late Night with Stephen Colbert show page: 1-2 seconds
Same tests, but while the PC and SickRage are more or less idle:
- Refresh (F5) the Daily Show with "show all seasons" ON: 14 to 18 seconds
- id. with option OFF: 10 to 14 seconds
- Refresh (F5) the Late Night with Stephen Colbert show page: 1 second

... seems that whether the option is on or off, displaying or refreshing a show page which has many seasons takes a considerable amount of time. I must say it's much less problematic in Chrome, which doesn't tend to freeze as easily as Firefox does (I mean: in Firefox, the whole browse freezes, while in Chrome, it's just the SickRage tab).
